config/saml.yml

```diff
@@ -1,9 +1,11 @@
 development: &defaults
-  issuer: saml-rp.vetsgov.localhost
-  callback_url: http://localhost:3001/auth/login/callback
+  issuer: saml-rp.vetsgov.encrypted.localhost
+  callback_url: http://localhost:3000/auth/saml/callback
   metadata_url: https://api.idmelabs.com/saml/metadata
   certificate: <%= File.read(File.expand_path(ENV["CERTIFICATE_FILE"])).dump %>
```

Comment:
Can we prefix the `CERTIFICATE_FILE`, `KEY_FILE`, and other SAML related vars? We've got a few different services that require various certificate configurations, and more of a distinction might make how they're applied clearer.